src/transactions/utils.ts

```diff
@@ -95,3 +103,55 @@ export const transferReviewHeader = (
 
   return headerText
 }
+
+export const filterTxByMonth = (
+  transactions: FeedTokenTransaction[],
+  month: string | undefined
+) => {
+  if (!month) return transactions
+  return filter(transactions, (tx) => {
+    const target = formatFeedSectionTitle(tx.timestamp, i18n)
+    return target === month
+  })
+}
```

Comment:
@Nashmod okay, seems that the screen is navigated with the filter 'Recent', but when the transactions are filtered, they are marked as 'August' so they get filtered out. I will take a look at this.

Reply:
Sounds good! Will wait on that update then.